content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -363,6 +363,38 @@ rules:
   verbs: ["get", "list", "watch"]
 ```
 
+#### Write Access for Endpoints
```

Comment:
nit:
```suggestion
#### Write access for Endpoints
```

Reply:
Updated, thanks!

Reply:
I think that's still using title case (we prefer sentence case). Only a nit anyway, mind.

Reply:
🤦 should be updated now, thanks!